gapic/src/main/com/google/gapid/models/Memory.java

```diff
@@ -618,7 +629,7 @@ private boolean mayHaveChildren() {
         case PSEUDONYM:
           TypeInfo.PseudonymType pseudonym = type.getPseudonym();
           childType = typesModel.getType(type(pseudonym.getUnderlying(), api));
-          children.add(new StructNode(api, childType, value, rootAddress, typesModel));
+          children.add(new StructNode(api, childType, value, rootAddress, typesModel, structName));
```

Comment:
Nit: Would prefer to use the previous way to initialize node (use the original constructor and setStructName() method).
Reason for suggestion: Use the constructor to initialize those final variables, but use has/get/set method for those very optional variables.

Reply:
Done.